x-pack/plugins/apm/public/services/rest/index_pattern.ts

```diff
@@ -12,3 +12,9 @@ export const createStaticIndexPattern = async () => {
     pathname: '/api/apm/index_pattern/static'
   });
 };
+
+export const getApmIndexPatternTitle = async () => {
+  return await callApmApi({
+    pathname: '/api/apm/index_pattern/title'
+  });
+};
```

Comment:
I don't think we want an endpoint for each config value the UI needs. Could we make this more generic so it returns all APM config values (supposing there is nothing confidential in them).

Reply:
There's some opportunity for that since we also get the various apm_oss indices for index settings default values. But Let's address it in another tech debt issue. I'd like to keep this PR as minimal as possible.

Reply:
Agree, I should have made it clear that this was for follow-up. We don't have to do it until the need arise (eg. we need to pass another config value to the UI)